src/current/cockroachcloud/resource-usage-basic.md

```diff
@@ -0,0 +1,191 @@
+
+[Connection pooling](https://www.cockroachlabs.com/docs/stable/connection-pooling) is the recommended way to manage the number of connections for many workloads. To read more about connection pooling, see our [What is Connection Pooling, and Why Should You Care](https://www.cockroachlabs.com/blog/what-is-connection-pooling/) blog post.
+
+### Excessive data egress
+
+In a CockroachDB {{ site.data.products.basic }} deployment, the {{ site.data.products.cloud }} Console does not provide direct observability of data egress, but you can observe the component of egress that comes from SQL Statements:
+
+1. Navigate to the [**Statements** tab]({% link cockroachcloud/statements-page.md %}) of your cluster's **SQL Activity** page in the {{ site.data.products.cloud }} Console.
+1. Sort the queries by the **Bytes Read** column. Most queries should read fewer than 1000 bytes.
+
+Excessive egress can be treated similarly to [expensive queries](#expensive-queries). Reducing the amount of data returned per query is often the best way to decrease egress data. You can also reduce the frequency of [excessive queries](#excessive-queries).
+
+### Database UI tools
+
+Database management tools like [DBeaver](https://dbeaver.com/) also consume RUs. They can cause excessive RU consumption by running expensive queries to populate views and periodically refreshing in the background if left running.
+
+To determine whether database UI tools are contributing to your RU usage, navigate to the **SQL Activity** page in the {{ site.data.products.cloud }} Console and search for queries similar to the following:
+
+~~~
+SELECT count(*) FROM crdb_internal.cluster_sessions
+
+SELECT avg((((statistics->'')->'')->'_')::INT8) AS meanrunlatency
+FROM crdb_internal.statement_statistics AS ciss
+WHERE ciss.aggregated_ts::DATE = current_date()
+~~~
+
+You might also see [multiple open connections](#excessive-number-of-connections) to your cluster that persist regardless of workload. This can indicate that multiple team members have a database UI tool running.
+
+### Data migration
+
+Initial data ingestion during a migration may consume a high number of RUs. Generally, optimized performance will also coincide with optimized RU consumption in this case. Refer to our [Migration Overview](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/migration-overview) documentation for more information.
+
+### Changefeeds (CDC)
+
+In a CockroachDB {{ site.data.products.basic }} deployment, the {{ site.data.products.cloud }} Console does not provide direct observability of changefeeds, but they can contribute to significant RU usage. Refer to our documentation on [Optimizing changefeeds](https://www.cockroachlabs.com/docs/stable/cdc-queries) for performance guidance that may decrease RU consumption. CockroachDB {{ site.data.products.advanced }} users can also [Monitor and Debug Changefeeds](https://www.cockroachlabs.com/docs/stable/monitor-and-debug-changefeeds) in the DB Console.
```

Comment:
It doesn't seem necessary to mention Advanced here. Alternatively, maybe use an info callout for that part?

```suggestion
In a CockroachDB {{ site.data.products.basic }} deployment, the {{ site.data.products.cloud }} Console does not provide direct observability of changefeeds. However, changefeeds can contribute to significant RU usage. Refer to [Change Data Capture Queries](https://www.cockroachlabs.com/docs/stable/cdc-queries) for performance guidance that may decrease RU consumption.
```

Reply:
updated